packages/ia-topnav/README.md

```diff
@@ -0,0 +1,122 @@
+# \<ia-topnav>
+
+The top navigation menu for archive.org
+
+## Installation
+
+```bash
+yarn add @internetarchive/ia-topnav
+```
+
+## Usage
+
+```js
+// ia-top-nav.js
+import IATopNav from '@internetarchive/ia-topnav';
+export default IATopNav;
+```
+
+```html
+<!-- index.html -->
+<script type="module">
+  import './ia-topnav.js';
+</script>
+
+<style>
+  /* Defaults */
+  ia-topnav {
+    --white: #fff;
+    --grey13: #222;
+    --grey20: #333;
+    --grey28: #474747;
+    --grey60: #999;
+    --grey80: #ccc;
+
+    --linkColor: #428bca;
+    --linkHoverColor: var(--white);
+    --subnavLinkColor: #aaa;
```

Comment:
QUESTION: Why were the `--linkColor` and `--subnavLinkColor` values not given color names above, so that `--linkColor` and `--subnavLinkColor` could be set to them?

Reply:
Also applies down below in `packages/ia-topnav/src/styles/ia-topnav.js`.

Reply:
The blue color was one that was only used once and I had no idea what specific type of blue it is. The subnav one is because it's not used more than once, so I didn't see the need for setting a variable to set a variable.